neo/Core/Blockchain.cs

```diff
@@ -466,6 +528,49 @@ protected void OnPersistCompleted(Block block)
             PersistCompleted?.Invoke(this, block);
         }
 
+        protected void ProcessAccountStateDescriptor(StateDescriptor descriptor, DataCache<UInt160, AccountState> accounts, DataCache<ECPoint, ValidatorState> validators, MetaDataCache<ValidatorsCountState> validators_count)
+        {
+            UInt160 hash = new UInt160(descriptor.Key);
+            AccountState account = accounts.GetAndChange(hash, () => new AccountState(hash));
+            switch (descriptor.Field)
+            {
+                case "Votes":
+                    Fixed8 balance = account.GetBalance(GoverningToken.Hash);
+                    foreach (ECPoint pubkey in account.Votes)
+                    {
+                        ValidatorState validator = validators.GetAndChange(pubkey);
```

Comment:
Maybe a good option would be to check null before, `if (validator == null)`

Reply:
It can't be `null` if it has been voted.